doc/source/reference/random/legacy.rst

```diff
@@ -4,27 +4,27 @@
 
 Legacy Random Generation
 ------------------------
-The `~mtrand.RandomState` provides access to
```

Comment:
Shouldn't this be corrected to :class:`~numpy.random.mtrand.RandomState` (ot be a link) instead of italicized?

Reply:
Same comment in all places that have `~mtrand.Randomstate`

Reply:
Any the others too that start `~

Reply:
I thought that :class: was needed to get linking.  I guess not.